stacks_queues/queue.py

```diff
@@ -23,34 +23,40 @@ def enqueue(self, element):
             In the store are occupied
             returns None
         """
-        pass
+        if self.is_full(): 
+            raise QueueFullException
+        return self.items.insert_at_end(element)
```

Comment:
`items` and `insert_at_end` aren't attributes or methods of the `Queue` class that we provide. 

Additionally, it looks like you are implementing a Queue with an array or linked list of some sort, but this project asks you to implement a Queue using a circular buffer.